input/pagecontent/4_examples.md

```diff
@@ -1,4 +1,9 @@
-### TestCases
+### Test Cases
+This section lists the test cases that are generated for the [CDS Level 3](https://www.hl7.org/fhir/uv/cpg/STU2/documentation-approach-06-01-levels-of-knowledge-representation.html){:target="_blank"} (L3) logical pathways for colorectal cancer guidelines for screening and management. The test case templates are generated in advance of developing computable L3 logic that implements the CDS L2 logical pathways. The test cases provide unit tests, both negative and positive, for each of the clinical logic pathways. These test cases consist of synthetic electronic health records for a hypothetical patient as well as the corresponding expected CDS outputs. Listed below are the synthetic electronic health record snippets for each of these test patient fixtures. See the [“Testing and Validation” Section](#cds-testing--validation){:target="_blank"} for more details. 
```

Comment:
```suggestion
This section lists the test cases that are generated for the [CDS Level 3](https://www.hl7.org/fhir/uv/cpg/STU2/documentation-approach-06-01-levels-of-knowledge-representation.html){:target="_blank"} (L3) logical pathways for colorectal cancer guidelines for screening and management. The test case templates are generated in advance of developing computable L3 logic that implements the CDS L2 logical pathways. The test cases provide unit tests, both negative and positive, for each of the clinical logic pathways. These test cases consist of synthetic electronic health records for a hypothetical patient as well as the corresponding expected CDS outputs. Listed below are the synthetic electronic health record snippets for each of these test patient fixtures. See the [“Testing and Validation” Section](3_specification.html#cds-testing--validation){:target="_blank"} for more details. 
```

Testing and Validation link should be updated to point to specific section within Specification page.

Reply:
I addressed this in commit https://github.com/CancerScreeningCDS/crcsm-cds/pull/13/commits/5a419918aa6c556ca17b40d1b0c891663fe9ba9c.